# The-Concordance-Between-RNA-Seq-and-Microarray-Data-Depends-on-Chemical-Treatment-and-Transcript-Abundance
Reproducing The Concordance Between RNA-Seq and Microarray Data Depends on Chemical Treatment and Transcript Abundance study by Wang et. al

Both RNA-seq and microarray are two known methods that are used to study
and measure gene expression. The purpose of the Wang et. al study was to compare
RNA-seq and microarray analysis. This was done by characterizing the concordance of
differential gene expression across platforms, test and compare how effective each
platform is at detecting expected pathway-level effects based on each treatments MOA,
and assess the MOA prediction accuracy of each platform using a test set [1]. One of
the advantages of RNA-seq is that it allows unbiased genome-wide gene-expression
profiling. However, its concordance with the microarray must be meticulously assessed
in order to be used in clinical application. RNA-seq is also superior to microarray, in
regards to differentially expressed genes (DEG) verification by quantitative PCR. For
low expressed genes, RNA-seq has improved accuracy as compared to microarray
analysis. However, when determining predictive classifiers, both RNA-seq and
microarray had similar performance. As a result, the intended application of
transcriptomic research is essential for method decision-making. The findings of Wang
et. al. determined that the cross-platform concordance with differentially expressed
genes or enriched pathways is highly correlated with treatment effect size, geneexpression
abundance and the biological complexity of the mode of action (MOA).
Using similar analysis methods, the purpose of our study is to replicate Wang et.
results [1]. One distinction to make is that in our study, a majority of the analysis was
conducted using precomputed data.
